charts/logging/loki/test/default.yaml

```diff
@@ -11,14 +11,3 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
```

Comment:
I assume this file is still required so the `test-chart-rendering.sh` script is being triggered. If so I think it makes sense to add a "This file is required by test-chart-rendering.sh. Do not delete it!", because if someone deletes the file later in some sort of "cleanup repo", it would not be picked up by CI.

Reply:
Fair point, will add that